docs/_documentations/codewind-eclipse-quick-guide.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: docs
```

Comment:
This doc seems to be for local Codewind, do we need it for the Minikube guide?

Reply:
Good point. Since quick guides are new, they don't have their own layout yet like `docs` and `guides` do. So, let me ask the larger ID team. I believe we want it laid out more like a doc, since it's quicker and shorter than a guide, but I'll confirm!